tooling/github-configuration.md

```diff
@@ -0,0 +1,103 @@
+# GitHub InnerSource Configuration
+
+- [Enterprise Setting](#enterprise-setting)
+  - [Repository policies](#repository-policies)
+    - [Base permissions](#base-permissions)
+    - [Repository creation](#repository-creation)
+    - [Repository forking (Private / Internal)](#repository-forking-private--internal)
+- [Organization Setting](#organization-setting)
+  - [Member privileges](#member-privileges)
+    - [Base permissions](#base-permissions-1)
+    - [Repository creation](#repository-creation-1)
+    - [Repository forking](#repository-forking)
+- [GitHub Enterprise Server Setting](#github-enterprise-server-setting)
+  - [GitHub Connect - Server statistics](#github-connect---server-statistics)
+- [Resources](#resources)
+
+## Enterprise Setting
+
+### Repository policies
+
+#### Base permissions
+
+When setting up the base repository permissions to promote InnerSource within your organization on GitHub, it's important to strike a balance between transparency, collaboration, and respecting constraints and limitations. Here are some best practices for setting up repository permissions:
+
+| Permission | Description |
+| --- | --- |
+| No Policy | Inherit permissions from the organization or parent repository. Provides flexibility for projects with unique permission needs. |
+| No Permission | Not recommended if any of the repositories in that org might benefit from collaboration. Completely restricts access to the repository, hindering collaboration and knowledge sharing. Repository access requires finding and talking to org owners. |
+| Read | All organization members can view the repository's contents, promoting transparency and allowing employees to learn from each other's work. |
+| Write | Specific individuals or teams with the "Write" permission can make contributions to the repository. Grant to active InnerSource participants with relevant skills. |
```

Comment:
Thank you. This is a good point. The discussion here is not yet active.
And the document doesn't mention Repository Rules, which is a new feature of GitHub Enterprise, the new alternative to Branch Protection setting. This will need to be presented as well